packages/trace-viewer/src/ui/watchMode.tsx

```diff
@@ -36,7 +36,7 @@ import { toggleTheme } from '@web/theme';
 import { artifactsFolderName } from '@testIsomorphic/folders';
 import { settings } from '@web/uiUtils';
 
-let updateRootSuite: (config: FullConfig, rootSuite: Suite, progress: Progress) => void = () => {};
+let updateRootSuite: (config: FullConfig, rootSuite: Suite, progress: Progress | undefined) => void = () => {};
```

Comment:
When we have no progress, we render "Tests", that's after reload or before we ran any tests.